docs/v1.0/post-installation-guide.txt

```diff
@@ -0,0 +1,100 @@
+# Post Installation Guide
+
+The goal of this article is to provide a concise post-installation guide to new Fluentd users.
+It is assumed that you've installed Fluentd through td-agent package.
+
+## System Administration
+
+### Configuration File
+
+A clean installation leaves you a td-agent instance running on a sample configuration file. You can edit the configuration file located at:
+
+    /etc/td-agent/td-agent.conf
+
+After editing this file, you need to restart td-agent using `systemctl`:
+
+    $ sudo systemctl restart td-agent
+
+### Logging
+
+By default, td-agent writes its operation logs to the following file:
+
+    /var/log/td-agent/td-agent.log
+
+If you want to make td-agent more verbose, read the article ["Trouble Shooting"](trouble-shooting).
+
+## Connect to Other Services
+
+### How It Works
+
+In Fluentd, the most important part of data input/output is managed by plugins. Each plugin knows how to interface with a external endpoint and is responsible for managing a pipeline to convey data streams.
+
+Plugins are named with a certain convention. For example, if it receives data and interfacing with Elasticsearch, it's called `in_elasticsearch`. In the same way, if it publishes data and connects to MongoDB, it's called `out_mongo`.
+
+The following snippet is an example configuration, which uses `in_forward` plugin as an input source and `out_file` plugin as an output endpoint.
+
+```
+<source>
+  @type forward
+  port 9999
+</source>
+<match app.**>
+  @type file
+  path /var/log/app/data.log
+  compress gzip
+</match>
+```
+
+### Plugin Management
+
+Fluentd manages plugins as Ruby gems, but stores these gems in a separate directory from where normal Ruby gems reside.
+
+This is why you need to use a special program `td-agent-gem` to manage Fluentd plugins. For example, the following command allows you to install the plugin to connect S3 (which contains both `in_s3` and `out_s3`)
+
+     $ sudo td-agent-gem install fluent-plugin-s3
```

Comment:
`/usr/sbin/td-agent-gem` is better. sometimes `/usr/sbin` is not exist in `PATH`.

Reply:
Thanks! Also fixed by #501.